test/integration/admin_can_create_store_items_test.rb

```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class AdminCanCreateStoreItemsTest < ActionDispatch::IntegrationTest
+  test 'admin can modify their account data' do
+    @category = Category.create(title: 'Apparel')
+    admin = User.create(username: 'admin',
+                        password: 'password',
+                        role:      1)
+
+    ApplicationController.any_instance.stubs(:current_user).returns(admin)
+    visit admin_dashboard_index_path(admin)
+    click_link 'Add items to store'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.